### PR TITLE
Update file outdated korean docs in dev-1.16-ko.2.

### PR DESCRIPTION
**Pull request by ysyukr**
from #16608

**9 out of 11 files to be modified**

- [x] content/ko/docs/concepts/overview/working-with-objects/namespaces.md
- [x] content/ko/docs/concepts/workloads/controllers/deployment.md
- [x] content/ko/docs/concepts/workloads/pods/pod-lifecycle.md
- [x] content/ko/docs/contribute/_index.md
- [x] content/ko/docs/reference/kubectl/cheatsheet.md
- [x] content/ko/docs/setup/learning-environment/minikube.md
- [x] content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
- [x] content/ko/docs/tasks/tools/install-minikube.md

**Unmodified document and reason.**
- [x] content/ko/docs/concepts/workloads/controllers/cron-jobs.md
  > The change to the document is CronJob in Cronjob. There is nothing to change in the translation.
- [x] content/ko/docs/tutorials/clusters/apparmor.md
  > Modify in #16531

### CODE REVIEW

File: content/ko/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -161,8 +161,8 @@ kubelet은 실행 중인 컨테이너들에 대해서 선택적으로 세 가지
 컨테이너가 보통 `initialDelaySeconds + failureThreshold × periodSeconds` 이후에 기동된다면, 스타트업 프로브가 활성화 프로브와 같은 엔드포인트를 체크하도록 명시해야 한다. `periodSeconds`의 기본 값은 30s 이다.
 이 때 컨테이너가 활성화 프로브의 기본 값 변경 없이 기동되도록 하려면 `failureThreshold`를 충분히 높게 설정해주어야 한다. 그래야 데드락(deadlocks)을 방지하는데 도움이 된다.
 
-활성 프로브 및 준비성 프로브를 설정하는 방법에 대한 추가적인 정보는,
-[활성 프로브 및 준비성 프로브 설정하기](/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)를 참조하면 된다.
+활성, 준비성 및 시작 프로브를 설정하는 방법에 대한 추가적인 정보는,
```
Comment (gochist) [inline content/ko/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
활성, 준비성 및 스타트업 프로브를 설정하는 방법에 대한 추가적인 정보는,
```

기존 문서에서 "스타트업 프로브"라는 용어를 사용한 문서가 있습니다. 일관성을 위해 이하 변경 바랍니다.

**Comment by ysyukr** [comment]
@gochist 리뷰 감사합니다.
말씀해주신 부분 문서에 반영했습니다!

**Comment by gochist** [comment]
@ysyukr 감사합니다!
/lgtm
/approve

**Pull request merged**